src/games/strategy/triplea/delegate/BattleTracker.java

```diff
@@ -1139,6 +1139,25 @@ void fightBombingRaids(final IDelegateBridge delegateBridge, Supplier<Collection
     }
   }
 
+  public void fightAutoKills(final IDelegateBridge delegateBridge) {
+    // Kill undefended transports. Done here to remove potentially dependent sea battles below
+    for( final Territory t : getPendingBattleSites(false) ) {  // Loop through normal combats i.e. not bombing or air raid
```

Comment:
Do you not have intelliJ either? Eclipse and Intellij both have auto-formatters that will do the formatting for you. It is highly encouraged to use either one so that we don't have to even talk about formatting in code reviews. To some extent, we introduced those auto-formatters because most of us really hate even having the formatting conversation.

Lambda's are a good thing to learn, it's actually a simpler and less error prone construct than the for loop + if statements.
It would look something like this:
```
 getPendingBattleSites(false).stream()
      .map(territory -> getPendingBattle(t, false, BattleType.NORMAL))
      .filter(batler -> Match.allMatch(battle.getDefendingUnits(), Matches.UnitIsDefenselessTransport))
      .forEach(battle -> battle.fight(delegateBridge);
```